dev/com.ibm.ws.jsf.internal/bnd.bnd

```diff
@@ -45,12 +45,12 @@ Service-Component: \
     implementation:=com.ibm.ws.jsf.ee.WASMyFacesContainerInitializer; \
     provide:=javax.servlet.ServletContainerInitializer; \
     properties:="service.vendor=IBM,service.ranking:Integer=101", \
-  com.ibm.ws.jsf_3_0.injection.class.list.collaborator; \
+  com.ibm.ws.jsf.injection.class.list.collaborator; \
     implementation:=com.ibm.ws.jsf.shared.util.JSFInjectionClassListCollaborator ; \
     provide:='com.ibm.wsspi.webcontainer.collaborator.WebAppInjectionClassListCollaborator'; \
     immediate:=true; \
     properties:="service.vendor=IBM", \
-  com.ibm.ws.org.apache.myfaces_3_0_${replace;${bVersion};\\.;_}.ResourceProvider; \
+  com.ibm.ws.org.apache.myfaces_${replace;${bVersion};\\.;_}.ResourceProvider; \
```

Comment:
file name changes may not be necessary if the transformer handles that. (I think it does)

Reply:
Never mind. Transformation rules needed to be updated since I changed the file names.